corehq/apps/es/case_search.py

```diff
@@ -128,8 +128,16 @@ def get_child_cases(self, case_ids, identifier):
             queries.MUST,
         )
 
-    def sort_by_case_property(self, case_property_name, desc=False):
+    def sort_by_case_property(self, case_property_name, desc=False, sort_type=None, reset_sort=False):
```

Comment:
If you call this with `sort_type=None` but `reset_sort=False` - it will still reset the sort (line 145).  I think you could just set `reset_sort=reset_sort` in that first one, though you'd have to either switch the default or update existing callers to avoid changing behavior.

Reply:
I think that's more or less what I want, to keep the behavior of existing calls to the function. I see how it's misleading though, since it implies you can set `reset_sort` on its own, but as-is it wouldn't always have an effect.

The nicest answer here seemed like just removing it, since I'm never passing in a `reset_sort` anymore, instead setting _either_ a `commcare_sort` or a default sort block. https://github.com/dimagi/commcare-hq/pull/33621/commits/f27de5f29ec10f55b5bd20e291b93ed2ba79d4f5

Reply:
nice, even better